Nclient.html

```diff
@@ -23,56 +23,435 @@
   	</nav>
 
 				  
-	<div class="account">
-		<div class="container">
-			<form action="" id="">
+		<div class="account">
+			<div class="container">
+				<form action="" id="">
 
 				<div class="row">
 					<div class="col s12">
-						<span>Step One: Create Account Information</span>
+						<h5>Create Log In</h5>
 					</div>
 				</div>
 
-				 <div class="row valign-wrapper">
+				<div class="row valign-wrapper">
 
 			      <div class="col s12 m6">
 			      	<div class="input-field">
-		          	<input id="email" name="email" type="email" class="validate">
-		          	<label for="email">Email</label>
-				   		</div>
+		          		<input id="email" name="email" type="email" class="validate">
+		          		<label for="email">Email</label>
+				   	</div>
 
-				   		<div class="input-field">
+				   	<div class="input-field">
 				   			<input id="password" name="password" type="password" class="validate">
 				   			<label for="password">Password</label>
-				   		</div>
+				   	</div>
 
-				   		<div class="input-field">
+				   	<div class="input-field">
 				   			<input id="confirm" name="confirm" type="text" class="validate">
 				   			<label for="confirm">Confirm</label>
-				   		</div>
-			      </div>
+				   	</div>
+			      	</div>
 			   	
-			   		<div class="col s12 m6 center-align">
+			   	<div class="col s12 m6 center-align">
 							<h6>Are you a </h6>
-			   			<input type="radio" name="business">
-			   			<label for="business">Business</label>
-			   			<input type="radio" name="lawfirm">
-			   			<label for="lawfirm">Lawfirm</label>
+			   		<input type="radio" name="type" class="js-change-modal" data-modal="businessModal" name="business">
+			   		<label for="business">Business</label>
+			   		<input type="radio" name="type" class="js-change-modal" data-modal="lawfirmModal" name="lawfirm">
+			   		<label for="lawfirm">Lawfirm</label>
+				</div>
+					
+					</div>
+				</div>
+			</div>
+			
+
+
+		<a class="waves-effect waves-light btn modal-trigger center" href="#modal1"></a>
```

Comment:
there's no text inside this `a` tag. You should remove this if you're not using it